legacy/querycounter/querycounter.go

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package querycounter
+
+import "sync"
+
+// networkMonitor records the number of HTTP requests (queries) to GCR.
+type networkMonitor struct {
+	mutex   sync.Mutex
+	queries uint64
```

Comment:
Please rename this variable to `requests`.

Reply:
ack.